### PR TITLE
Add toggles

### PR DESCRIPTION
**Pull request by haoxins**

### CODE REVIEW

File: src/progress.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+const React = require('react');
+const mdl = require('material-design-lite/material.min');
+const classnames = require('classnames');
+
+const baseClasses = {
+  'mdl-progress': true,
+  'mdl-js-progress': true
+};
+
+class Loading extends React.Component {
```
Comment (phated) [inline src/progress.js]: change the class name to Progress

**Comment by phated** [comment]
Just 1 thing.  Excellent work

File: src/progress.js (hunk 1 above)
Comment (haoxins) [inline src/progress.js]: done

**Comment by phated** [comment]
Thanks! Published as 0.5.0

**Pull request merged**